project/plugins.sbt

```diff
@@ -13,7 +13,6 @@ val sbtProtocV = "1.0.6"
 
 buildInfoKeys := Seq[BuildInfoKey]("sbtProtocVersion" -> sbtProtocV)
 
-addSbtPlugin("lt.dvim.authors" % "sbt-authors" % "1.3")
```

Comment:
This plugin adds the ability to generate a list of authors from git log which was Lightbends/akkas method of generating release notes. Due to it relying on an older binary incompatible version of Circe (something which was fixed with latest pekko-sbt-paradox) we can no longer use this plugin.

```
[error] java.lang.RuntimeException: found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error]
[error] 	* io.circe:circe-core_2.12:0.14.6 (early-semver) is selected over {0.13.0}
[error] 	    +- com.github.sbt:sbt-paradox-material-theme:0.7.0 (scalaVersion=2.12, sbtVersion=1.0) (depends on 0.14.6)
[error] 	    +- io.circe:circe-jawn_2.12:0.13.0                    (depends on 0.13.0)
[error] 	    +- io.circe:circe-generic_2.12:0.13.0                 (depends on 0.13.0)
[error]
[error]
[error] this can be overridden using libraryDependencySchemes or evictionErrorLevel
[error] 	at scala.sys.package$.error(package.scala:30)
[error] 	at sbt.internal.LibraryManagement$.resolve$1(LibraryManagement.scala:90)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$12(LibraryManagement.scala:134)
[error] 	at sbt.util.Tracked$.$anonfun$lastOutput$1(Tracked.scala:74)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$20(LibraryManagement.scala:147)
[error] 	at scala.util.control.Exception$Catch.apply(Exception.scala:228)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11(LibraryManagement.scala:147)
[error] 	at sbt.internal.LibraryManagement$.$anonfun$cachedUpdate$11$adapted(LibraryManagement.scala:128)
[error] 	at sbt.util.Tracked$.$anonfun$inputChangedW$1(Tracked.scala:220)
[error] 	at sbt.internal.LibraryManagement$.cachedUpdate(LibraryManagement.scala:161)
[error] 	at sbt.Classpaths$.$anonfun$updateTask0$1(Defaults.scala:3801)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:63)
[error] 	at sbt.std.Transform$$anon$4.work(Transform.scala:69)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:283)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:24)
[error] 	at sbt.Execute.work(Execute.scala:292)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:283)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:265)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:65)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
[error] 	at java.lang.Thread.run(Thread.java:750)
[error] (update) found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error]
[error] 	* io.circe:circe-core_2.12:0.14.6 (early-semver) is selected over {0.13.0}
[error] 	    +- com.github.sbt:sbt-paradox-material-theme:0.7.0 (scalaVersion=2.12, sbtVersion=1.0) (depends on 0.14.6)
[error] 	    +- io.circe:circe-jawn_2.12:0.13.0                    (depends on 0.13.0)
[error] 	    +- io.circe:circe-generic_2.12:0.13.0                 (depends on 0.13.0)
```

Since we no longer do this (we have removed all references of running this plugin) its easiest to just remove it for now. I have created an upstream issue https://github.com/2m/authors/issues/415 if it gets fixed and/or we decide to use it again